homeassistant/components/seventeentrack/repairs.py

```diff
@@ -0,0 +1,76 @@
+"""Repairs for the SeventeenTrack integration."""
+
+from typing import cast
+
+import voluptuous as vol
+
+from homeassistant.components.repairs import ConfirmRepairFlow, RepairsFlow
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers import issue_registry as ir
+
+from . import DOMAIN
+from .const import DEPRECATED_KEY
+
+
+@callback
+def deprecate_sensor_issue(hass: HomeAssistant, entry_id: str) -> None:
+    """Ensure an issue is registered."""
+    ir.async_create_issue(
+        hass,
+        DOMAIN,
+        "deprecate_sensor",
+        breaks_in_ha_version="2025.1.0",
+        issue_domain=DOMAIN,
+        is_fixable=True,
+        is_persistent=True,
+        translation_key="deprecate_sensor",
+        severity=ir.IssueSeverity.WARNING,
+        data={"entry_id": entry_id},
+    )
```

Comment:
Can we move this to the place where we create it? We don't create it at many places